js/public/script.js

```diff
@@ -1129,6 +1129,19 @@ app.service('vCardPropertiesService', [function() {
 			readableName: t('contacts', 'Role'),
 			template: 'text'
 		},
+		cloud: {
+			multiple: true,
+			readableName: t('contacts', 'Federated Cloud Id'),
```

Comment:
Only script.js is loaded by the Browser. This file is concatenated from all the others.